lib/streamlit/elements/utils.py

```diff
@@ -64,65 +64,37 @@ def _build_duplicate_widget_message(
     return message.strip("\n").format(widget_type=widget_func_name, user_key=user_key)
 
 
-def _set_widget_id(
-    element_type: str,
-    element_proto: Any,
-    user_key: Optional[str] = None,
-    widget_func_name: Optional[str] = None,
-) -> None:
-    """Set the widget id.
-
-    Parameters
-    ----------
-    element_type : str
-        The type of the element as stored in proto.
-    element_proto : proto
-        The proto of the specified type (e.g. Button/Multiselect/Slider proto)
-    user_key : str or None
-        Optional user-specified key to use for the widget ID.
-        If this is None, we'll generate an ID by hashing the element.
-    widget_func_name : str or None
-        The widget's DeltaGenerator function name, if it's different from
-        its element_type. Custom components are a special case: they all have
-        the element_type "component_instance", but are instantiated with
-        dynamically-named functions.
+def get_widget_id(
```

Comment:
It doesn't look like this is used outside `utils.py`, should we prefix with an underscore?

Reply:
Ah - yeah, that works for me. (I'm actually using it from within the `st.form` PR I'm working on, but I can make it public there instead.)

Reply:
Oh -- that also makes sense, you didn't have to change it here then! But it looks like you already did :P